### PR TITLE
Video details page

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
Video details page created

### CODE REVIEW

**Review by Shubhamkr2610 — APPROVED** [review]
looks good to me , nothing suggestion from my side . you can go for merge

File: src/components/cards/VideoCard.js [hunk 1]
```diff
@@ -8,9 +8,10 @@ export const VideoCard = ({
   chanel,
   views,
   createdAt,
+  id,
 }) => {
   return (
-    <div className="video-card">
+    <Link to={`/watch/${id}`} className="video-card">
       <div className="thumbnail">
         <img src={thumbnail} alt="thumnail" />
```
Comment (Jia-A) [inline src/components/cards/VideoCard.js]: you might wanna change the spelling of thumbnail in alt.

File: src/pages/videoDetailsPage/videoDetails.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  display: block;
+  margin:  1rem 0;
+}
+.title-wrapper p{
+  color: var(--primary-inverse);
+}
+.player-bottom{
+  height: fit-content;
+  padding-bottom: 1rem;
+  border-bottom: 1px solid var(--primary-inverse-20);
```
Comment (Jia-A) [inline src/pages/videoDetailsPage/videoDetails.css:42]: Good use of CSS variables.

**Review by Jia-A — APPROVED** [review]
Everything looks good.
You can format your code with Beautify or Prettier.
Good to merge.

**Pull request merged**